hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServlet.java

```diff
@@ -439,7 +442,9 @@ public void testWriteDbDataToStream() throws Exception {
 
     // Get the tarball.
     when(responseMock.getOutputStream()).thenReturn(servletOutputStream);
+    long tmpHardLinkFileCount = tmpHardLinkFileCount();
     omDbCheckpointServletMock.doGet(requestMock, responseMock);
+    Assertions.assertEquals(tmpHardLinkFileCount, tmpHardLinkFileCount());
```

Comment:
It is not necessarily always 0. I.e. if bootstrapping was run before change this PR introduced it is likely not to be 0. Because of that this assertion confirms there are no more tmp files created than there already are.